stubs/draft.stub

```diff
@@ -0,0 +1,20 @@
+models:
```

Comment:
At the most, the stub should make a `models` and `controllers` block. Maybe only this `# ...` underneath.

Otherwise, an empty file is fine.

Reply:
Truncated draft.stub